net/mosquitto/Makefile

```diff
@@ -146,7 +147,7 @@ endef
 define Package/libmosquittopp
     SECTION:=libs
     CATEGORY:=Libraries
-    DEPENDS:=libmosquitto +libstdcpp
+    DEPENDS:=libmosquitto $(CXX_DEPENDS)
```

Comment:
this _appears_ to be completely unrelated to "compile without openssl deprecated apis"

Reply:
I added a note in the commit message.

Reply:
I'm still not ever going to merge this under the title of "compile without open ssl deprecated apis"  this looks by all means a valid fix, btu it's a completely separate commit.

Reply:
also, does the $(CXX_DEPENDS) work to allow selecting the library directly? or do you now have to go and select a c++ lib first?

Reply:
Ok. I will place it under a separate commit sometime in the future.